ffi/CMakeLists.txt

```diff
@@ -30,7 +33,7 @@ set(CMAKE_REQUIRED_INCLUDES ${LLVM_INCLUDE_DIRS})
 CHECK_INCLUDE_FILES("llvm/IR/SVML.inc" HAVE_SVML)
 if(HAVE_SVML)
     message(STATUS "SVML found")
-    add_definitions(-DHAVE_SVML)
+    add_definitions(-DHAVE_SVML -DLLVM_ENABLE_PROJECTS="lld")
```

Comment:
If `-DLLVM_ENABLE_PROJECTS="lld"` is required, it shouldn't be conditional on the presence of SVML, which is unrelated (it's the "Short Vector Math Library", I believe) - addition of this definition should take place outside this conditional for SVML.

Reply:
Makes sense. I will change this when I get home.